ql/src/java/org/apache/hadoop/hive/ql/ddl/misc/msck/MsckAnalyzer.java

```diff
@@ -68,7 +68,8 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
     }
 
     Table table = getTable(tableName);
-    Map<Integer, List<ExprNodeGenericFuncDesc>> partitionSpecs = ParseUtils.getFullPartitionSpecs(root, table, conf, false);
+    Map<Integer, List<ExprNodeGenericFuncDesc>> partitionSpecs = ParseUtils.getFullPartitionSpecs(root, table, conf,
+        false);
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

Reply:
Removed.